src/hoc/withTheme.js

```diff
@@ -93,6 +93,10 @@ const wrapWithTheme = (Component: ReactClass<any>) => {
     }
   }
 
+  WithTheme.propTypes = {
```

Comment:
This is also missing from the `StyledComponent` itself. So either we don't add it at all, or at least we add it to everything in a separate PR, I'd say, so we can discuss it there :wink:

Reply:
Yeah I've noticed it wasn't described anywhere. The problem is the the pre-commit run eslint which fire an error if it's not provided.